tests/integration/conftest.py

```diff
@@ -43,6 +43,10 @@ async def charmed_kubernetes(ops_test):
             deploy, control_plane_app = False, control_plane_apps[0]
 
         if deploy:
+            # vv - work-around for LP#1968849
+            await model.set_config({"lxd-snap-channel": "4.24/stable"})
+            # ^^ - once lxd 5.0/stable & juju 2.9/stable work together, use this workaround
+
```

Comment:
this may only be necessary for a few days while waiting on https://github.com/juju/juju/pull/13957 to get released into `juju latest/stable`